website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

```diff
@@ -0,0 +1,146 @@
+
+Data Vault follows the insert only principle with incremental loading strategy. A built-in **jinja** functionality allows you to create one version of the dbt model for both incremental and full load of a table. The easy dependency management that this feature helps you achieve is a huge benefit for highly complex projects.
+
+If you are new to the framework, taking a look at already built Data Vault macros can be crucial, and even if you are an expert, it can still be beneficial. dbt’s rich set of community **packages** can be directly applied to your project or used as an inspiration for your own transformation templates.
+
+Building your transformation templates leveraging reusable macros and flexible jinja language can enhance your project development in a scalable way. When things get more complex, you are able to go back and change your templates in one place either completely, or using parameters to ensure you don’t mess with what already works well.
+
+Scalability also happens at the database layer. With **materialisations**, you have fine-grained control over whether a database object built by dbt is persisted as a view, table, or built incrementally, which gives you control over the performance and cost characteristics of each transformation. So if your data platform bill is growing, it’s easy to identify which Data Vault components are the most expensive, and make optimisations to reduce cost.
+
+With the active dbt open source community, there is a good chance you are facing a problem which was already solved by someone else. There are plenty of amazing packages available in the dbt [package hub](https://hub.getdbt.com/), which you can utilise to speed up your development even further.
+
+### Agile development
+
+dbt Cloud includes **built-in Git** with accessible features directly from its IDE, which simplifies development immensely. Once a developer is happy with their additions or changes to the Data Vault codebase, they can commit the code within the IDE and open a Pull Request, triggering a code review process. Then, with [continuous integration with dbt Cloud](https://docs.getdbt.com/docs/deploy/continuous-integration), automated checks are run to ensure data quality standards and Data Vault conventions are met, automatically preventing any bad changes from reaching production. 
+
+The biggest boon to Data Vault developer productivity in dbt Cloud are the **DataOps** and **Data Warehouse Automation** features of dbt Cloud. Each Data Vault developer gets their own development environment to work in and there is no complicated set up process to go through.
+
+Commit your work, create a penabled by **jobs** that can be defined for each environment separately and then scheduled or triggered by a specific event. Together with dbt **tags**, the feature allows you to orchestrate your project in an efficient and powerful way.
```

Comment:
what do you mean by 'penabled'? do you mean 'create and enable **jobs**,  which you can define for each environment..